src/AboutPage/About.tsx

```diff
@@ -1,8 +1,28 @@
-import React from 'react'
-
 function About() {
   return (
-    <div>About</div>
+    <>
+      <section className='flex justify-center items-center p-10 flex-col'>
+        <h1 className=' text-5xl m-5 md:m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-screen'/>
```

Comment:
Horizontal padding should match the rest of the page content.
Current:
<img width="1512" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/b229bfea-966b-44cb-9f67-e7f4bdc38e1f">
Intended:
<img width="726" alt="image" src="https://github.com/webimpactuw/poke-bay/assets/25410985/7d65eace-42b5-437c-91f4-54952462f263">